lib/actions/apiV2.js

```diff
@@ -1000,7 +1000,7 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
       unpreferred,
       // TODO: Does this break everything?
       ...currentQuery,
-      numItineraries: config?.modes?.numItineraries || numItineraries || 7
+      numItineraries: numItineraries || config?.modes?.numItineraries || 7
```

Comment:
@miles-grant-ibigroup This is to give preference to the URL parameter if it is specified, instead of the config. Is that a conflict with #1232?

Reply:
Also, should we make the default number of itineraries 3 across the board?

Reply:
I've checked and this works fine! I think 7 as a default is fine as well

Reply:
I am confused why this can't just use the function you wrote to generate this default? A little worried we're going to look at this in the future and be really confused what is happening with `numItineraries`. Why would we want 7 here but a 3 where the URL parameters are generated?

Reply:
@miles-grant-ibigroup Was 7 the default for the old batch UI? In the dev branch, it gets overwritten by the `numItineraries=3` param every time, so we could use 3 here too.

Reply:
I think 7 is the default for the new batch UI too. 3 is an override for when using combinations

Reply:
Is there a deployment where 7 itineraries returned is the desired outcome with the default settings?

Reply:
@daniel-heppner-ibigroup Thoughts?

Reply:
After discussion with @daniel-heppner-ibigroup and further investigation, it turns out `numItineraries` that is passed in `currentQuery` is initialized from the initial state of `state.otp.currentQuery`, which is set upon creation of the otp (and user) reducer, and updated as needed from the `numItineraries` URL param, if set. In theory, `currentQuery.numItineraries` is always defined. So, the default of 7 is in theory never hit. Just to be sure, I'll set the fallback to be consistent with the other cases by using `getDefaultNumItineraries(config)`.